cloud/blockstore/libs/storage/disk_registry/disk_registry_state.cpp

```diff
@@ -6695,16 +6729,19 @@ auto TDiskRegistryState::QueryAvailableStorage(
             continue;
         }
 
-        const ui64 au = GetAllocationUnit(device.GetPoolName());
-
-        ++chunks[au];
+        if (DeviceList.IsDirtyDevice(device.GetDeviceUUID())) {
+            auChunks.DirtyChunks++;
+        } else if (!DeviceList.IsAllocatedDevice(device.GetDeviceUUID())) {
+            auChunks.FreeChunks++;
+        }
+        auChunks.ChunkCount++;
     }
 
     TVector<TAgentStorageInfo> infos;
     infos.reserve(chunks.size());
 
-    for (auto [size, count]: chunks) {
-        infos.push_back({ size, count });
+    for (auto [size, counts]: chunks) {
```

Comment:
Не понял почему counts
```suggestion
    for (auto [size, info]: chunks) {
```